data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/_treasury_ot_pharaohs.pcc

```diff
@@ -0,0 +1,26 @@
+# CVS $Revision a.1$ $Author Axxroy$ -- Sun Feb 25 14:45:00 2017 -- reformated by prettylst.pl v1.35 (build 553)
+CAMPAIGN:Legendary - Treasury of the Pharaohs
+GAMEMODE:Pathfinder
+RANK:9
```

Comment:
Improper rank here, you need to go lower. Ranks 7, 8, and 9 are all Homebrew ranks, For a 3rd party, the ranks are 4, 5 and 6.  4 is a Core Rule set, 5 is supplements, and 6 would be accessory supplements.